packages/mobile/src/components/lineup-tile/LineupTile.tsx

```diff
@@ -68,15 +71,16 @@ export const LineupTile = ({
   const premiumConditions = isTrack ? item.premium_conditions : null
   const isArtistPick = artist_pick_track_id === id
   const { doesUserHaveAccess } = usePremiumContentAccess(isTrack ? item : null)
-  const dispatch = useDispatch()
 
   const showPremiumDogEar =
     premiumConditions &&
     (isOwner || !doesUserHaveAccess) &&
     !(showArtistPick && isArtistPick)
 
   const dogEarType = showPremiumDogEar
-    ? premiumConditions.nft_collection
+    ? isUSDCEnabled && premiumConditions.usdc_purchase
```

Comment:
I assume you're following up with changes to use the helper function?

Reply:
yes